package.json

```diff
@@ -21,6 +21,7 @@
     "pregenerate": "npm-run-all api:bundle*",
     "generate": "npm-run-all generate:*",
     "generate:definitions-internal": "rimraf ./src/generated/definitions/internal && shx mkdir -p ./src/generated/definitions/internal && gen-api-models --api-spec ./api/internal.yaml --no-strict --out-dir ./src/generated/definitions/internal ",
+    "generate:definitions-fn-lollipop": "rimraf ./src/generated/definitions/fn-lollipop && shx mkdir -p ./src/generated/definitions/fn-lollipop && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/io-functions-lollipop/master/openapi/external.yaml --no-strict --out-dir ./src/generated/definitions/fn-lollipop --request-types --response-decoders --client",
```

Comment:
`generate:client-fn-lollipop` is more appropriate

Reply:
Fixed in 52e77cb